utils/requestOptions.js

```diff
@@ -0,0 +1,17 @@
+const { X_RAPIDAPI_URL, X_RAPIDAPI_HOST, X_RAPIDAPI_KEY } = process.env;
+
+export const requestOptions = (cityName) => {
```

Comment:
better use opt, it allows easy to add any parameters

Reply:
If I understood correctly, i moved all consts from .env to config file